packages/airnode-examples/src/scripts/choose-integration.ts

```diff
@@ -3,7 +3,7 @@ import { readdirSync, writeFileSync } from 'fs';
 import { join } from 'path';
 import { PromptObject } from 'prompts';
 import { goSync } from '@api3/airnode-utilities';
-import { cliPrint, IntegrationInfo, promptQuestions, runAndHandleErrors } from '../src';
+import { cliPrint, IntegrationInfo, promptQuestions, runAndHandleErrors } from '../';
```

Comment:
I know importing from `../` is awkward, but it's fundamentally not wrong, so I left it like this (instead of importing from the individual files).